### DISCUSSION TITLE
Bot not taking customizations

### DISCUSSION THREAD

**Discussion by connorI55** [jef/streetmerchant#1248] [Help] [answered]
## Description

I'm sure I'm just doing something wrong, but I can't get the bot to take my customization file.  I update it to only look for 3080's, and the rx6900xt, and from a few select stores.  But for some reason it's just using the default settings.

Any ideas?
<img width="617" alt="Screen Shot 2020-12-07 at 10 41 30 PM" src="https://user-images.githubusercontent.com/75655165/101437152-91cd2280-38dd-11eb-9aaa-bc2f70f7adba.png">
<img width="299" alt="Screen Shot 2020-12-07 at 10 40 45 PM" src="https://user-images.githubusercontent.com/75655165/101437156-9265b900-38dd-11eb-8729-d168e17064be.png">
<img width="558" alt="Screen Shot 2020-12-07 at 10 40 36 PM" src="https://user-images.githubusercontent.com/75655165/101437157-9265b900-38dd-11eb-8e69-954d0b220b16.png">

**Comment by zethussuen** [comment, marked as answer]
turn on your ability to view file extensions (https://www.howtogeek.com/205086/beginner-how-to-make-windows-show-file-extensions/) and you'll see that your `.env` file is actually `.env.env-example`. The script isn't picking up your `.env` because it's not actually there. You'll need to remove the file .env-example file extension so that it's just a `.env` file.

**Comment by connorI55** [comment]
@zethussuen that was it.  I knew it was just me, lol.  Thanks a ton!
